### DISCUSSION TITLE
DRM with OvenPlayer

### DISCUSSION THREAD

**Discussion by antonymarion** [OvenMediaLabs/OvenMediaEngine#1663] [Help]
Hi all,

I am using OvenPlayer and Ome (both with Docker) and love it!

But I can not see the DRM part on UI here:
![image](https://github.com/user-attachments/assets/17298289-108c-49da-b5c9-2e72ba3985ab)

Is it normal?

I saw some screenshot on this forum of the player but with the DRM part.

**Comment by getroot** [comment]
The OvenPlayer Docker Image is for demo/testing purposes only.
Try https://demo.ovenplayer.com/ or http://demo.ovenplayer.com. To test DRM, you will also need to install the certificate on your OME server.
